### PR TITLE
Update quick-search-box homepage

### PR DESCRIPTION
**Pull request by scribblemaniac**

### CODE REVIEW

File: Casks/quick-search-box.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   url "https://qsb-mac.googlecode.com/files/GoogleQuickSearchBox-#{version}.Release.dmg"
   name 'Quick Search Box'
-  homepage 'http://www.google.com/quicksearchbox/'
+  homepage 'https://code.google.com/p/qsb-mac/'
```
Comment (vitorgalvao) [inline Casks/quick-search-box.rb]: This does not sound like a good idea, since google code will be shut down. Is there no other URL?

Comment (scribblemaniac) [inline Casks/quick-search-box.rb]: None that I could find. The old link is broken right now so this is at least a temporary improvement. I'm pretty sure this project is dead though, considering it hasn't release anything in 5 years, so the cask could probably just get removed when google code shuts down.

Comment (vitorgalvao) [inline Casks/quick-search-box.rb]: Agreed.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as eba4f27f87c487f67ae5d782a68acdb348e43254 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**